website/templates/website/components/navbar.html

```diff
@@ -1,34 +1,43 @@
-<nav class="navbar navbar-expand-lg navbar-dark sticky-top">
-    <div class="container">
-        <a class="navbar-brand" href="{% url 'home' %}">RedCraft<span class="text-white-50">.org</span></a>
-        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
-            <span class="navbar-toggler-icon"></span>
-        </button>
+{% load static %}
 
-        <div class="collapse navbar-collapse flex-row-reverse" id="navbarSupportedContent">
-            <ul class="navbar-nav flex-row-reverse">
-                <li class="nav-item {% if page == 'home' %}active{% endif %}">
-                    <a class="nav-link" href="{% url 'home' %}">Accueil</a>
-                </li>
-                <li class="nav-item {% if page == 'vote' %}active{% endif %}">
-                    <a class="nav-link" href="{% url 'vote' %}">Vote</a>
-                </li>
-                <li class="nav-item {% if page == 'dons' %}active{% endif %}">
-                    <a class="nav-link" href="{% url 'dons' %}">Dons</a>
-                </li>
-                <li class="nav-item {% if page == 'stats' %}active{% endif %}">
-                    <a class="nav-link" href="{% url 'stats' %}">Stats</a>
-                </li>
-                <li class="nav-item {% if page == 'rules' %}active{% endif %}">
-                    <a class="nav-link" href="{% url 'rules' %}">Règles</a>
-                </li>
-                <li class="nav-item {% if page == 'dynmap' %}active{% endif %}">
-                    <a class="nav-link" href="{% url 'dynmap' %}">Dynmap</a>
-                </li>
-                <li class="nav-item {% if page == 'contact' %}active{% endif %}">
-                    <a class="nav-link" href="{% url 'contact' %}">Contact</a>
-                </li>
-            </ul>
+<nav class="navbar">
+    <div class="container flex-column flex-lg-row align-items-lg-center">
+        <input type="checkbox" id="navbar-toggler">
+        <div class="navbar-top">
+            <a class="navbar-brand" href="{% url 'home' %}">
+                <img class="d-inline-block" src="{% static 'website/img/inline_org_color.png' %}" alt="Redcraft.org logo">
+            </a>
+            </button>
+            <label for="navbar-toggler" class="toggler-icon">
+                <div class="icon"></div>
+            </label>
+        </div>
+        <div class="navbar-collapse">
+            <div class="container justify-content-lg-end">
+                <ul class="navbar-nav flex-column flex-lg-row-reverse mt-4 mt-lg-0">
+                    <li class="nav-item {% if page == 'home' %}active{% endif %}">
+                        <a class="nav-link" href="{% url 'home' %}">Accueil</a>
```

Comment:
Not sure who originally wrote this code but I think this should be a for loop instead of copy/pasted stuff